wordpress-mobile/WordPress-iOS
# Activity Log: UI updates

This:
* updates how the AL displays status of a restore currently in progress 
* gets rid of HUD in AL in favor of Notices
* makes sure that we display date/time in site's timezone, not UTC

To test:

* TZ stuff:
    *  Make sure that your website is set to a different timezone than UTC
    * Go to AL
    * Go to any event details
    * Verify that the date/time displayed is correct (you can use Calypso to compare if you're not keen on doing TZ math in your head :P)
* Restore status stuff
    * Go to AL
    * Start a restore
    * Verify the notices posted at the bottom of the screen make sense
    * Verify that there's a new section at the top of the AL during the restore process
    * Verify that the new section displays progress at least somewhat in sync with Calypso (we poll the network much less often than Web does to save our user's data, but it should be ~roughly the same).

Fun fact: during debugging this I found out that I have a site that has a restore hanging since *drumroll* April, 4th. Given it reports as 78% percent complete, it should be done somewhere around July 23rd :P
